code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -104,11 +139,18 @@
 /datum/chemical_reaction/reagent_explosion/penthrite_explosion_epinephrine
 	required_reagents = list(/datum/reagent/medicine/c2/penthrite = 1, /datum/reagent/medicine/epinephrine = 1)
 	strengthdiv = 5
+	/// Penthrite is rare as hell, so this clears your reagents
+	/// Will most likely be from miners accidentally penstacking
+	clear_mob_reagents = TRUE
+
 
 /datum/chemical_reaction/reagent_explosion/penthrite_explosion_atropine
 	required_reagents = list(/datum/reagent/medicine/c2/penthrite = 1, /datum/reagent/medicine/atropine = 1)
 	strengthdiv = 5
 	modifier = 5
+	/// Rare reagents clear your reagents
+	/// Probably not good for you because you'll need healing chems to survive this most likely
```

Comment:
```suggestion
	// Penthrite is rare as hell, so this clears your reagents
	// Will most likely be from miners accidentally penstacking
	clear_mob_reagents = TRUE


/datum/chemical_reaction/reagent_explosion/penthrite_explosion_atropine
	required_reagents = list(/datum/reagent/medicine/c2/penthrite = 1, /datum/reagent/medicine/atropine = 1)
	strengthdiv = 5
	modifier = 5
	// Rare reagents clear your reagents
	// Probably not good for you because you'll need healing chems to survive this most likely
```